dask/array/tests/test_linalg.py

```diff
@@ -449,35 +449,22 @@ def test_dask_svd_self_consistent(m, n):
         assert d_e.dtype == e.dtype
 
 
-@pytest.mark.slow
 def test_svd_compressed():
-    m, n = 2000, 250
-    r = 10
-    np.random.seed(4321)
-    mat1 = np.random.randn(m, r)
-    mat2 = np.random.randn(r, n)
-    mat = mat1.dot(mat2)
-    data = da.from_array(mat, chunks=(500, 50))
-
-    u, s, vt = svd_compressed(data, r, seed=4321, n_power_iter=2)
-
-    usvt = da.dot(u, da.dot(da.diag(s), vt))
-
-    tol = 0.2
-    assert_eq(
-        da.linalg.norm(usvt), np.linalg.norm(mat), rtol=tol, atol=tol
-    )  # average accuracy check
+    m, n = 100, 50
+    r = 5
+    a = da.random.random((m, n), chunks=(m, n))
 
-    u = u[:, :r]
-    s = s[:r]
-    vt = vt[:r, :]
+    # calculate approximation and true singular values
+    u, s, vt = svd_compressed(a, 2 * r, seed=4321, n_power_iter=0)  # worst case
+    s_true = scipy.linalg.svd(a.compute(), compute_uv=False)
 
-    s_exact = np.linalg.svd(mat)[1]
-    s_exact = s_exact[:r]
+    # compute the difference with original matrix
+    norm = scipy.linalg.norm((a - (u[:, :r] * s[:r]) @ vt[:r, :]).compute(), 2)
 
-    assert_eq(np.eye(r, r), da.dot(u.T, u))  # u must be orthonormal
-    assert_eq(np.eye(r, r), da.dot(vt, vt.T))  # v must be orthonormal
-    assert_eq(s, s_exact)  # s must contain the singular values
+    # ||a-a_hat||_2 <= (1+tol)s_{k+1}
+    frac = norm / s_true[r + 1] - 1
+    tol = 0.4
```

Comment:
Can you explain where this comes from in a comment?  Linking to https://github.com/dask/dask/pull/6799#issuecomment-725631268 would be helpful.

Reply:
Do you mean putting it in a comment in the code itself? Or do you mean here on github?

Reply:
I did mean in the code itself, maybe something like this?

```suggestion
    # Tolerance determined via simulation to be slightly above max norm of difference matrix in 10k samples.
    # See https://github.com/dask/dask/pull/6799#issuecomment-725631268 for more details.
    tol = 0.4
```